bin/post_compile

```diff
@@ -80,21 +97,23 @@ heroku-install-cabot-plugins() {
   done
 }
 
+heroku-put-scripts-on-path() {
+  # A better way of doing this perhaps?
+  ln -sf /app/.heroku/vendor/node-4.2.6/bin/* /app/.heroku/python/bin/
+}
+
 heroku-compress-assets() {
   puts-step "Compressing assets"
-  PATH=$PATH:/app/$NODE_PATH/bin .heroku/python/bin/python manage.py compress | indent
+  PATH=$PATH:/app/$NODE_PATH/bin .heroku/python/bin/python manage.py compress --force | indent
 }
 
 
 main() {
-  if [[ ! -n $DYNO ]]; then
```

Comment:
$DYNO is heroku-specific. I figured I could update this to *also* check for a flynn env var, but then I wondered if we could just do without it entirely. Presumably this script should only be run from a buildpack-compatible system anyway?